lintro/ai/invoke.py

```
@@ -0,0 +1,105 @@
+            max_retries=ai_config.max_retries,
+            base_delay=ai_config.retry_base_delay,
+            max_delay=ai_config.retry_max_delay,
+            backoff_factor=ai_config.retry_backoff_factor,
+        )
+        def _call() -> AIResponse:
+            return _call_once()
+
+    try:
+        return cast(AIResponse, _call())
+    except AIAuthenticationError:
+        raise
```

Ruff (0.15.20)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)